tests/scripts/bulk_import.py

```diff
@@ -68,9 +68,20 @@ def do_import_mem(resource, dataset):
 
 
 def do_create():
+    class _BookResource(resources.ModelResource):
```

Comment:
What is the reason for duplicating the declaration in this method?

Reply:
Since it's only creating brand new objects I am setting the appropriate flags for performance:
skip_diff to True
removing cacheInstanceLoader
force_init_instance to True

Reply:
thanks - did you try running the benchmark without these changes in, i.e only your change in `resources.py`?

Reply:
Of course, without any changes (hence on 10000 rows)

**before changes**
do_create()
Time   2.487
Memory 2.62890625
Max memory 78.21875

**after changes**
do_create()
Time   2.186
Memory 1.65234375
Max memory 72.37890625

Reply:
And with custom BookResource:
do_create()
Time   0.8369
Memory 1.73828125
Max memory 65.64453125